### DISCUSSION TITLE
Feature suggestion

### DISCUSSION THREAD

**Discussion by Dal78** [ClassicOldSong/Apollo#124] [Ideas]
Feature suggestion:   Virtual mirror window.

Fantastic fork. The virtual display works much better to avoid all the frame sync resolution issues.

Might i suggest that a future feature could be that when a virtual display is invoked a mirror window, which is just a framebuffer dump + scale is displayed within a window on the main monitor.  

This is the approach VR headsets use and is a nice quality of life feature without comprimising the virtual display frametimes.  Not intended for gameplay just useful.

**Comment by ClassicOldSong** [comment]
You can already do that by launching Moonlight on the same computer and pair with the same host.
